app/components/AddText/index.js

```diff
@@ -0,0 +1,23 @@
+import styled from 'styled-components';
+
+const colors = {
```

Comment:
It might be a good idea to move shared colors to separate vairables file

Reply:
decided